### PR TITLE
add simple test move data

### PR DESCRIPTION
**Pull request by datanel**
- added a test for tartare.tasks.update_data_task
- added documentation

### CODE REVIEW

File: requirements_dev.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-r requirements.txt
```
Comment (kinnou02) [inline requirements_dev.txt]: nice!

**Comment by antoine-de** [comment]
we need to explain how the tests are ran

File: tests/update_data_test.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import os
+from tartare.tasks import move_data
+
+def test_update_data_move_files(tmpdir):
```
Comment (antoine-de) [inline tests/update_data_test.py]: I really like when there is a comment on each test (even if there it's a bit obvious)

**Comment by antoine-de** [comment]
we also need to say we use python 3.4

File: tests/update_data_test.py (hunk 2 above)
Comment (datanel) [inline tests/update_data_test.py]: done

File: README.md [hunk 3]
```diff
@@ -1,2 +1,33 @@
 # tartare
 data integration
+
```
Comment (antoine-de) [inline README.md]: :raised_hands:

File: README.md [hunk 4]
```diff
@@ -1,2 +1,33 @@
...
+## Requirements
+- python 3.4
+- [RabbitMQ](https://www.rabbitmq.com/)
+
+## Installation
+
+You can use [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/) for creating virtual environments.
+
+```
+cd path/to/tartare
```
Comment (antoine-de) [inline README.md]: you don't need to be in the directory to interact with your virtualenv

File: README.md [hunk 5]
```diff
@@ -1,2 +1,33 @@
...
+```
+
+Installation of dependencies
+```
+pip install -r requirements.txt
+```
+
+## Run the application
+```
+pip install honcho
```
Comment (antoine-de) [inline README.md]: maybe we can put honcho in the requrements_dev.txt

what do you think @kinnou02 ?

Comment (datanel) [inline README.md]: In the doc I also use honcho for running the app and honcho is optional

File: README.md (hunk 4 above)
Comment (pbougue) [inline README.md]: yep, so you can move that line just before pip  install

**Comment by pbougue** [comment]
ok for me

**Pull request merged**